### PR TITLE
Fix Jaeger to work again in helm.

### PR DESCRIPTION
**Pull request by edwarnicke**
## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: k8s/cmd/admission-webhook/const.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ const (
 	repoEnv              = "REPO"
 	initContainerEnv     = "INITCONTAINER"
 	tagEnv               = "TAG"
+	tracerEnabledEnv     = "TRACER_ENABLED"
```
Comment (nickolaev) [inline k8s/cmd/admission-webhook/const.go]: Shall we add these to `docs/env.md`?

File: deployments/helm/nsm/values.yaml [hunk 2]
```diff
@@ -25,4 +25,4 @@ spire:
 
 global:
   # set to true to enable Jaeger tracing for NSM components
-  JaegerTracing: false
+  JaegerTracing: true
```
Comment (nickolaev) [inline deployments/helm/nsm/values.yaml]: Hmm I would suggest this to be false by default and for development purposes to enable it in the `make helm-install-nsm` target.

File: deployments/helm/icmp-responder/templates/nsc.tpl [hunk 3]
```diff
@@ -5,7 +5,7 @@ spec:
   selector:
     matchLabels:
       networkservicemesh.io/app: "icmp-responder-nsc"
-  replicas: 4
+  replicas: 1
```
Comment (nickolaev) [inline deployments/helm/icmp-responder/templates/nsc.tpl]: Really depends what we want to show with this simple example. 1 replica might be enough.

File: k8s/cmd/admission-webhook/const.go (hunk 1 above)
Comment (edwarnicke) [reply k8s/cmd/admission-webhook/const.go]: It is already at the top of docs/env.md ... but that said it defaulted to True.  Fixed that in latest push :)

File: deployments/helm/nsm/values.yaml (hunk 2 above)
Comment (edwarnicke) [reply deployments/helm/nsm/values.yaml]: Good idea. Fixed.

File: deployments/helm/icmp-responder/templates/nsc.tpl (hunk 3 above)
Comment (edwarnicke) [reply deployments/helm/icmp-responder/templates/nsc.tpl]: This should never have been in the PR, was a tweak I did for local development.

**Pull request merged**